src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -2304,6 +2304,110 @@ class InsertStatementSegment(BaseSegment):
     )
 
 
+@ansi_dialect.segment()
+class MergeStatementSegment(BaseSegment):
+    """A `MERGE` statement."""
+
+    type = "merge_statement"
+    match_grammar = StartsWith(
+        Sequence("MERGE", "INTO"),
```

Comment:
For BigQuery (and T-SQL) the `INTO` is optional. Wonder if we should:
1. Override in those dialects
2. Have the `INTO` optional in ANSI
3. Define as a Grammar to make it easier to override in other dialects without having to copy and paste basically the whole grammar?

Think I'm veering towards 3. WDYT?

Reply:
I agree with three.
One and two seem like workarounds.

Reply:
done